### PR TITLE
add lbc image, fix wallet balance

### PR DESCRIPTION
**Pull request by lemsmyth**
#42 gets available wallet balance adds LBC icon to available wallet amount

### CODE REVIEW

**Comment by lyoshenka** [comment]
thanks for the PR @lemsmyth! @prkirby could you please take a look at this when you get a chance

would love to show you both some [appreciation](https://lbry.com/faq/appreciation) for your contribution :-)

File: classes/LBRY_Daemon.php [hunk 1]
```diff
@@ -69,15 +69,17 @@ public function address_list($page = 1)
...
      */
     public function wallet_balance()
     {
-        try {
-            $result = $this->request('account_balance');
-            return $result->result->available;
+        try { // Convert JSON string to an object
+            $result = $this->request('wallet_balance');
+            return $result;
+            $wallet_balance = $result;
```
Comment (prkirby) [inline classes/LBRY_Daemon.php:82]: This statement isn't reachable it looks like? should it be above the return statement?

Comment (prkirby) [reply classes/LBRY_Daemon.php:82]: I'm assuming this is just updating the call? Where is $wallet_balance used?

**Review by prkirby — CHANGES_REQUESTED** [review]
Just one comment

File: classes/LBRY_Daemon.php (hunk 1 above)
Comment (lemsmyth) [reply classes/LBRY_Daemon.php]: Yes you are correct not reachable and unneeded, have removed the $wallet_balance variable and also updated the try - catch to throw updated error message. 
Using the $wallet_balance lines 3-4 and 11 on options-page to drill down and access result->available to output the available, unstaked amount in the wallet, prior was showing a much smaller amount (maybe the amount from one address?) than was actually available for use in the wallet.

**Review by prkirby — APPROVED** [review]
LGTM!

**Comment by lyoshenka** [comment]
thanks y'all!

**Pull request merged**